docs/monitoring/monitoring-troubleshooting.asciidoc

```diff
@@ -22,3 +22,28 @@ The following error appears in a banner at the top of the screen in {kib} on the
 You cannot monitor a version 6.3 or later cluster from {kib} version 6.2 or earlier. 
 To resolve this issue, upgrade {kib} to 6.3 or later. See 
 {stack-ref}/upgrading-elastic-stack.html[Upgrading the {stack}]. 
+
+[float]
+=== No monitoring data is visible in {kib}
+
+*Symptoms:*
+
+The *Monitoring* page in {kib} is empty. 
+
+*Resolution:*
+
+. Confirm that {kib} is seeking monitoring data from the appropriate URL. By 
```

Comment:
Should we say "appropriate Elasticsearch URL" or is that implied at this point in the doc?

Reply:
Done!